item_list.php

```diff
@@ -1,14 +1,12 @@
+<!-- Fn -->
 <?php
+  
+  include_once('check_session.php');
   include_once('header.php');
   include_once('database.php');
-  session_start();
-  if (!isset($_SESSION) || $_SESSION == null) {
-    echo('<div class="text-center">You\'re not logged in. Please re-login if this was a mistake</div>');
-    header('refresh:3;url=browse.php');
-  }
 
   $user_id = $_SESSION['id'];
-  $items_query = "SELECT Item.id, Item.name, Item.description, Item.image_url, Category.name as category_name FROM Item JOIN Category ON Item.category_id = Category.id WHERE user_id = $user_id;";
+  $items_query = "SELECT Item.id, Item.name, Item.description, Item.image_url, Category.name as category_name FROM Item JOIN Category ON Item.category_id = Category.id WHERE user_id = $user_id AND is_available = TRUE;";
```

Comment:
This will only list the items that are currently not a part of any auction.
If we only want the user to be able to remove items that are not listed in any auction, we can add validation checks for the same in the UI component where we show the button, but ideally we should be listing all the items that the user has ever created.